pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -306,6 +329,28 @@ func (r *ReconcileClusterProvision) reconcileRunningJob(instance *hivev1.Cluster
 	return reconcile.Result{}, nil
```

Comment:
If it has been less than `podStatusCheckDelay`, need to requeue with a delay equal to the time remaining from `podStatusCheckDelay`. Otherwise, the ClusterProvision will not be reconciled again in a reasonable time to set the InstallPodStuckCondition.